docs/README.md

```diff
@@ -0,0 +1,59 @@
+# Psoxy Documentation
+
+## FAQs
+
+ - [Security](faq-security.md)
+
+### Host Environments
+
+  - [AWS](aws/getting-started.md)
+  - [GCP](gcp/getting-started.md)
+
+
+### Deployment Environments
+Psoxy can be *hosted* in GCP or AWS, but it can be *deployed* to those host platforms from a variety
+of locations.
+
+  - local machine (macOS or a Linux such as Ubuntu likely to be most straightforward options)
+  - remote VM/container (eg, EC2 instance)
+  - CI/CD, such as GitHub Actions, Atlassian Bamboo, etc.
+  - [Terraform Cloud / Terraform Enterprise](terraform-cloud.md)
+  - Cloud Shell: [aws](aws/cloud-shell.md) (not recommended) / [gcp](gcp/cloud-shell.md)
```

Comment:
just double checking: is aws cloud shell not recommended or both (aws and gcp) ? Also, I'd add a comment about why we don't recommend this approach here